core-bundle/src/Resources/config/services.yml

```diff
@@ -921,6 +921,8 @@ services:
             - '@Contao\CoreBundle\Twig\Loader\ContaoFilesystemLoader'
         public: true
 
+    Contao\CoreBundle\Twig\Interop\ContextFactory: ~
```

Comment:
With regards to #3686 and upstream mergeability, this should be `contao.twig.context_factory`.

Reply:
Changed in 815b891.